cli/src/config/merge_tools.toml

```diff
@@ -52,10 +52,13 @@ merge-args = ["--wait", "--merge", "$left", "$right", "$base", "$output"]
 # markers. Unfortunately, it does not seem to be able to output conflict markers when
 # the user only resolves some of the conflicts.
 merge-tool-edits-conflict-markers = true
+# VS Code prefers Git-style conflict markers
+conflict-marker-style = "git"
```

Comment:
https://github.com/martinvonz/jj/pull/4897#discussion_r1853311074

> Set "snapshot" for vimdiff? I don't use vimdiff but I'm pretty sure I've heard that vimdiff users would prefer that. Maybe some vimdiff user can confirm?

Reply:
This makes sense to me. I think it gives better diff highlighting, since it allows vim to find more matching lines.